### ISSUE TITLE
tried loading the plugin but got this message

### ISSUE THREAD

**Issue by theunsfvo** [gimatdinov/gimatdinov.archiannotations#1]
![image](https://user-images.githubusercontent.com/5095077/38021878-4dd848e2-327e-11e8-9e28-406ff6d2f235.png)

**Comment by gimatdinov** [comment]
OS: Windows, Linux or Mac?
Arch: x32 or x64?

**Comment by theunsfvo** [comment]
Win10 pro, 64x

Thanks,
Theuns

*IQ Business <http://www.iqbusiness.net/>: Solution Architect | TOGAF 9
Certified Enterprise Architect  *
*"I love deadlines. I like the whooshing sound they make as they fly by." *
*  - Douglas Adams*
*    My 3D Images <http://bluestarcreations.wordpress.com/> || Some of my
writings <http://www.writingroom.com/writingresults.aspx?m=Zandria> || My
Awesome Stuff Blog <http://awesomestufftheunslikes.wordpress.com/about/>*

**Comment by gimatdinov** [comment]
Archi Installation x32 or x64?
See https://www.archimatetool.com/download
Use x64, on x32 not tested

**Comment by theunsfvo** [comment]
i'm sure i do have 64x installed but will download and make sure


Thanks,
Theuns

*IQ Business <http://www.iqbusiness.net/>: Solution Architect | TOGAF 9
Certified Enterprise Architect  *
*"I love deadlines. I like the whooshing sound they make as they fly by." *
*  - Douglas Adams*
*    My 3D Images <http://bluestarcreations.wordpress.com/> || Some of my
writings <http://www.writingroom.com/writingresults.aspx?m=Zandria> || My
Awesome Stuff Blog <http://awesomestufftheunslikes.wordpress.com/about/>*

**Comment by theunsfvo** [comment]
nope, installed version Archi-Win64-4.2 and added the plug but got the same
error as before...

Thanks,
Theuns

*IQ Business <http://www.iqbusiness.net/>: Solution Architect | TOGAF 9
Certified Enterprise Architect  *
*"I love deadlines. I like the whooshing sound they make as they fly by." *
*  - Douglas Adams*
*    My 3D Images <http://bluestarcreations.wordpress.com/> || Some of my
writings <http://www.writingroom.com/writingresults.aspx?m=Zandria> || My
Awesome Stuff Blog <http://awesomestufftheunslikes.wordpress.com/about/>*

**Comment by gimatdinov** [comment]
Problem with preference page only?
Is Annotations work?
Try create property  with name $ in any diagram element.

**Comment by theunsfvo** [comment]
properties annotation doesn't work either...




Thanks,
Theuns

*IQ Business <http://www.iqbusiness.net/>: Solution Architect | TOGAF 9
Certified Enterprise Architect  *
*"I love deadlines. I like the whooshing sound they make as they fly by." *
*  - Douglas Adams*
*    My 3D Images <http://bluestarcreations.wordpress.com/> || Some of my
writings <http://www.writingroom.com/writingresults.aspx?m=Zandria> || My
Awesome Stuff Blog <http://awesomestufftheunslikes.wordpress.com/about/>*

**Comment by gimatdinov** [comment]
Is exist any errors with my plugin in $userhome/AppData/Roaming/Archi4/.metadata/.log ?

**Comment by theunsfvo** [comment]
Thanks,
Theuns

*IQ Business <http://www.iqbusiness.net/>: Solution Architect | TOGAF 9
Certified Enterprise Architect  *
*"I love deadlines. I like the whooshing sound they make as they fly by." *
*  - Douglas Adams*
*    My 3D Images <http://bluestarcreations.wordpress.com/> || Some of my
writings <http://www.writingroom.com/writingresults.aspx?m=Zandria> || My
Awesome Stuff Blog <http://awesomestufftheunslikes.wordpress.com/about/>*

**Comment by gimatdinov** [comment]
Perhaps the problem is related to the fact that the plugin is downloaded from the Internet and windows marked the plugin file.
Try right click the Archi.exe and choose "Run As Administrator". 
I thought about it when I read this https://developer.salesforce.com/forums/?id=906F00000008o2YIAQ

**Comment by theunsfvo** [comment]
tried it, no luck... got it installed as portable in a local user folder,
not the normal windows program files, specifically because of admin
permission issues.




Thanks,
Theuns

*IQ Business <http://www.iqbusiness.net/>: Solution Architect | TOGAF 9
Certified Enterprise Architect  *
*"I love deadlines. I like the whooshing sound they make as they fly by." *
*  - Douglas Adams*
*    My 3D Images <http://bluestarcreations.wordpress.com/> || Some of my
writings <http://www.writingroom.com/writingresults.aspx?m=Zandria> || My
Awesome Stuff Blog <http://awesomestufftheunslikes.wordpress.com/about/>*

**Comment by gimatdinov** [comment]
I refactored plugin initialization, please, try the new version 0.0.6.

**Comment by theunsfvo** [comment]
thank you, will give it a try on Tuesday when i'm back at the office and
let you know...

Thanks,
Theuns

*IQ Business <http://www.iqbusiness.net/>: Solution Architect | TOGAF 9
Certified Enterprise Architect  *
*"I love deadlines. I like the whooshing sound they make as they fly by." *
*  - Douglas Adams*
*    My 3D Images <http://bluestarcreations.wordpress.com/> || Some of my
writings <http://www.writingroom.com/writingresults.aspx?m=Zandria> || My
Awesome Stuff Blog <http://awesomestufftheunslikes.wordpress.com/about/>*
